statler_api/api_models.py

```diff
@@ -42,29 +45,42 @@ def __init__(self, dao=None, text=None, date=None):
 
         # Bind DAO fields to the new instance.
         self.text = dao.text
-        # TODO: Resolve naming discrepancy
-        self.date = dao.timestamp
-
-# TODO: Work in progress.
-# class NewReview:
-#     """Represents a new review, prior to being saved in the database."""
-#
-#     # TODO: Figure out automatic serialization from the JSON request.
-#
-#     def __init__(self, playUrlTitle, text):
-#         """Construct a new review from posted text"""
-#
-#         self.text = text
-#         self.playUrlTitle = playUrlTitle
-#
-#         # timestamp the new review right now.
-#         self.timestamp = time.time()
-#
-#     def toDao(self):
-#         dao = ReviewDAO()
-#
-#         dao.text = self.text
-#         dao.timestamp = self.timestamp
+        self.timestamp = dao.timestamp
```

Comment:
NOTE: This is an API change. I should probably make the requisite spec and javascript changes.